stackit/internal/services/iaas/publicip/resource.go

```diff
@@ -151,7 +151,7 @@ func (r *publicIpResource) Schema(_ context.Context, _ resource.SchemaRequest, r
 				},
 			},
 			"network_interface_id": schema.StringAttribute{
-				Description: "Associates the public IP with a network interface or a virtual IP (ID).",
+				Description: "Associates the public IP with a network interface or a virtual IP (ID). If you are using the public_ip resource with an SKE Load Balancer or any other resource which associates a network interface implicitly, use the lifecycle `ignore_changes` property in this field to prevent unintentional removal of the network interface due to drift in the terraform state",
```

Comment:
```suggestion
				Description: "Associates the public IP with a network interface or a virtual IP (ID). If you are using this resource with a Kubernetes Load Balancer or any other resource which associates a network interface implicitly, use the lifecycle `ignore_changes` property in this field to prevent unintentional removal of the network interface due to drift in the Terraform state",
```